Formula/pulumi.rb

```diff
@@ -5,7 +5,7 @@ class Pulumi < Formula
       tag:      "v3.10.3",
       revision: "d8ebde1590088a21216329b429a83f7b39b512ef"
   license "Apache-2.0"
-  head "https://github.com/pulumi/pulumi.git"
+  head "https://github.com/pulumi/pulumi.git", branch: "master"
```

Comment:
I think we default head build to master branch, unless it is something else.

Reply:
i did this due to

```console
❯ brew audit --strict --online pulumi
pulumi:
  * HEAD: Use `branch: "master"` to specify the default branch
Error: 1 problem in 1 formula detected
```

looks like this was added in https://github.com/Homebrew/brew/pull/11720

Reply:
I see, thanks!